modules/fixtures/policies/principal_policy.tmpl

```diff
@@ -91,7 +91,6 @@
         "codestar:*",
         "cognito-identity:*",
         "cognito-idp:*",
-        "cognito-sync:*",
```

Comment:
@nathanagood what's the story behind removing some of the services in this policy?

Reply:
Cognito Sync doesn't appear anywhere in the AWS Nuke resources code. SimpleDB does, but it's an old, deprecated service and use of it should be avoided compared to other services (DDB).